outputs.tf

```
@@ -1,25 +1,25 @@
 # CloudTrail
 output "cloudtrail_id" {
   description = "The name of the trail"
-  value       = join("", aws_cloudtrail.this.*.id)
+  value       = aws_cloudtrail.this.id
 }
 
 output "cloudtrail_home_region" {
   description = "The region in which the trail was created"
-  value       = join("", aws_cloudtrail.this.*.home_region)
+  value       = aws_cloudtrail.this.home_region
 }
 
 output "cloudtrail_arn" {
   description = "The Amazon Resource Name of the trail"
-  value       = join("", aws_cloudtrail.this.*.arn)
+  value       = aws_cloudtrail.this.arn
 }
 
 output "log_group" {
   description = "The CloudWatch log group object created when no previous log group is declared"
-  value       = length(aws_cloudwatch_log_group.this) > 0 ? aws_cloudwatch_log_group.this[0] : null
+  value       = aws_cloudwatch_log_group.this
```

This resource is still optional, so this actually changes the output format. Instead of the returning the object (e.g. `{...}`), it returns a list(object) (e.g. `[{...}]`). But we should change the condition to match the condition used by the resource...

```suggestion
  value       = local.create_log_group ? aws_cloudwatch_log_group.this[0] : null
```